virtcontainers/cnm.go

```diff
@@ -19,6 +24,10 @@ func cnmLogger() *logrus.Entry {
 
 // init initializes the network, setting a new network namespace for the CNM network.
 func (n *cnm) init(config NetworkConfig) (string, bool, error) {
+	if config.NetNSPath == hostNetworkPath {
```

Comment:
Why do you need to introduce this hardcoded check ? You have a generic solution in `initNetworkCommon()`.

Reply:
I have got rid of the the docker specific check.